### DISCUSSION TITLE
2.34.2

### DISCUSSION THREAD

**Discussion by TheLastProject** [CatimaLoyalty/Android#2257] [Releases]
- Improve archive/starred icon display

<hr /><em>This discussion was created from the release <a href='https://github.com/CatimaLoyalty/Android/releases/tag/v2.34.2'>2.34.2</a>.</em>

**Comment by TheLastProject** [comment]
Hi @IzzySoft, @linsui,

I understand everyone is probably busy with the holidays, so please don't feel rushed. This release is mostly a small release to try to meet Google's consistently increasingly draconian requirements and contains a bugfix that should for only really affect users who archive/starred cards with certain custom/pkpass-sourced images.

To comply with Google's absurd rules without "ruining" the general Catima build, I've finally split Catima into two flavors: `gplay` and `foss`. Right now, there are no major differences, the gplay flavor just hides the Donate button (previously, this hiding was based on the installer app, but getting the installer is now deprecated). I am not planning on ever allowing the Google Play version of Catima to be "the best version". If anything, I really want people to use Google Play as little as possible :)

@IzzySoft Please note the binary added to releases is now named `app-foss-release.apk ` instead of  `app-release.apk`, in case this affects the IoD update checker. When checking for RB, please use the `foss` flavor.

@linsui Please make sure F-Droid builds the `foss` flavor. Please keep the "soft-RB-fail" check with SHA256SUMS, but make sure to compare your unsigned release to the `app-foss-release-unsigned.apk` entry.

**Reply by linsui** [reply]
Done. :ok_hand:

**Reply by TheLastProject** [reply]
I see Izzy did his part yesterday already too: https://floss.social/@IzzyOnDroid/113721791493452360, awesome :) Thank you both

**Reply by IzzySoft** [reply]
Yupp. Apologies for the late response here, but I was visiting family and thus could not respond earlier here.
